### PR TITLE
Silva references

### PR DESCRIPTION
**Pull request by aortiz24**
updated silva database and README.md

### CODE REVIEW

File: mothur_SilvaRef.bat [hunk 1]
```diff
@@ -5,16 +5,16 @@
 
 # rename downloaded database
```
Comment (k8hertweck) [inline mothur_SilvaRef.bat]: remove this chunk?

**Comment by k8hertweck** [comment]
This is great work! Two things to tweak:

1. You've deleted the two files from `silva/` that we actually need (the Ecoli files). Please add them back.
2. I would prefer to leave the directory name `silva` and include directions in the documentation to move `silva.nr_v128.align` to that directory.

**Comment by aortiz24** [comment]
I have implemented the changes you recommended. Let me know if there is something else that needs to be tweaked.

**Comment by k8hertweck** [comment]
This PR hadn't been reviewed and merged. Do you have a copy of the branch stored locally that you can restore, to reopen the PR?

**Pull request closed without merging**